algorithms/Algorithms_Answers.md

```diff
@@ -1 +1,47 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+#Excercise I.
+
+a.) Linear? The bigger n is, the bigger the loop grows. ```n*n*n``` is still n runtime.
+
+b.) log n. i is being split in half, then looping back through to check a value that is plit in 
+half; only to be split in half again.
+
+c.) First for loop would be log n/ 2, but then second is n, and 3rd is n. Time is as big as the biggest runtime so it would probably be ```n**2/ n ^ 2```.
+
+d.) Outside for loop is ```i < n``` but ```i*=2``` so I will double itself which means it gets to n, faster. Which would be log n. Nest for loop is n. So in total, ```n log n```?
+
+e.) First loop is n, second loop is n, 3rd loop is n, 4th loop is n or constant to 10?
```

Comment:
Yup, you're correct with the fourth loop being constant. Cubic — O(n³) is the correct answer.